app/views/blogs/edit.html.haml

```diff
@@ -0,0 +1,25 @@
+- if user_signed_in?
```

Comment:
Remove this validation since it is already authenticated in the blogs_controller before_filter. @jedilaunchlabs